roles/collect_modified_images/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# In order for this role to work, add the following task to your playbook
+# immediately before calling the deployment role.
+# - name: "Set the deployment start time"$a
+#   shell: "date -u '+%Y-%m-%dT%H:%M:%SZ'"
+#   register: start_time
+- name: "Create empty list for images"
+  set_fact:
```

Comment:
I think the definition of `pulled_images` and `events_file_path` should be moved to `roles/collect_modified_images/defaults/main.yml` since that file is currently empty and should be used for that purpose.